apps/server/src/modules/tool/controller/tool-school.controller.spec.ts

```diff
@@ -139,6 +139,57 @@ describe('ToolSchoolController', () => {
 		});
 	});
 
+	describe('getSchoolExternalTool is called', () => {
```

Comment:
We don't need controller tests. This should be replaced with API testing

Reply:
the API test is already done